tests/test_syncer.py

```diff
@@ -796,10 +800,32 @@ def test_shotgun_tag(self, mocked_sg):
         # The invalid tag should be reject, only the valid one should be there
         self.assertEqual(issue.fields.labels, ["foo"])
 
-    def test_jira_assignment(self, mocked_sg):
+    def test_hosted_jira_assignment(self, mocked_sg):
```

Comment:
We have to run this test twice now, once for testing the assignment code for JIRA Cloud and another for a local server.